clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtils.java

```diff
@@ -209,6 +209,19 @@ public String validateString(String name, boolean isRequired) throws ValidateExc
         return value;
     }
 
+    /**
+     * Validates that a value, if supplied, is a {@link Boolean}. If no value is present in the configuration, a
+     * default value of {@link Boolean#FALSE} is returned.
+     */
+    public Boolean validateBoolean(String name) {
+        Boolean value = get(name);
+
+        if (value != null)
+            return value;
+
+        return Boolean.FALSE;
```

Comment:
the other methods take argument `boolean isRequired`. maybe this method should follow the pattern?

Reply:
Done.